algos/aho_corasick/render_test.html

```diff
@@ -27,15 +30,55 @@
   </style>
   </head>
   <body>
-  <svg id="test1" width=960 height=600></svg>
+  <div id="test1">
+    <svg width="30em" height="18em"><g></g></svg>
+    <form onsubmit="return false">
+      <input type="text" autocomplete="off" id="keyword_text"></br>
+      <span id="keyword_span"></span>
+    </form>
+  </div>
 <script>
-
 var trie = new Trie();
-trie.addWord('abb');
-trie.addWord('abc');
-trie.addWord('abed');
-renderTrie(trie, "#test1");
+var cursor = trie;
 
+trie.renderTo("#test1", cursor);
```

Comment:
Are we still using this file if the trie is now static, generated from a dot graph file?